scripts/prepare_dutchf3.py

```diff
@@ -9,49 +9,59 @@
 
 
-def _get_splits_path(data_dir):
-    return path.join(data_dir, "splits")
-
-
-def _get_labels_path(data_dir):
-    return path.join(data_dir, "train", "train_labels.npy")
-
-
 def _write_split_files(splits_path, train_list, test_list, loader_type):
-    file_object = open(path.join(splits_path, loader_type + "_train_val.txt"), "w")
+    if not path.isdir(splits_path):
+        mkdir(splits_path)
+    file_object = open(path.join(splits_path,
+                       loader_type + "_train_val.txt"), "w")
     file_object.write("\n".join(train_list + test_list))
     file_object.close()
-    file_object = open(path.join(splits_path, loader_type + "_train.txt"), "w")
+    file_object = open(path.join(splits_path,
+                       loader_type + "_train.txt"), "w")
     file_object.write("\n".join(train_list))
     file_object.close()
-    file_object = open(path.join(splits_path, loader_type + "_val.txt"), "w")
+    file_object = open(path.join(splits_path,
+                       loader_type + "_val.txt"), "w")
     file_object.write("\n".join(test_list))
     file_object.close()
 
 
-def _get_aline_range(aline, per_val):
-    # Inline sections
-    test_aline = math.floor(aline * per_val / 2)
-    test_aline_range = itertools.chain(range(0, test_aline), range(aline - test_aline, aline))
-    train_aline_range = range(test_aline, aline - test_aline)
-
-    return train_aline_range, test_aline_range
+def _get_aline_range(aline, per_val, slice_steps):
+    try:
+        if slice_steps < 0:
+            raise ValueError('slice_steps cannot be a negative number')
+        # Inline and Crossline sections
+        test_aline = math.floor(aline * per_val / 2)
+        test_aline_range = itertools.chain(range(0, test_aline),
+                                           range(aline - test_aline, aline))
+        train_aline_range = range(test_aline, aline - test_aline, slice_steps)
+        return train_aline_range, test_aline_range
+    except (Exception, ValueError):
+        raise
 
 
-def split_section_train_val(data_dir, per_val=0.2, log_config=None):
+def split_section_train_val(data_dir, output_dir, label_file, per_val=0.2,
+                            log_config=None, slice_steps=1):
     """Generate train and validation files for Netherlands F3 dataset.
 
     Args:
         data_dir (str): data directory path
-        per_val (float, optional): the fraction of the volume to use for validation.
-            Defaults to 0.2.
+        output_dir (str): directory under data_dir to store the split files
+        label_file (str): npy files with labels. Stored in data_dir
+        stride (int): stride to use when sectioning of the volume
+        patch_size (int): size of patch to extract
+        per_val (float, optional):  the fraction of the volume to use for
+            validation. Defaults to 0.2.
+        log_config (str): path to log configurations
+        slice_steps (int): number of slices to be skipped. Defaults to 1.
```

Comment:
Probably more detailed comment on how\why to use slice_steps could be used.
Also if read this comment as is, I  may get an impression that by default we are always skipping 1 (always skipping every other slice) but by default we should NOT skip anything.

Reply:
Olga, I updated the documentation regarding that. 0 will not move the pointer and 1 means it will go to the next occurrence. 2 will skip every other slice.

The new message in doc is: "slice_steps (int): number of slices to be skipped. Defaults to 1, do not skip any slice."